lib/hocon/impl/parser.rb

```diff
@@ -807,6 +807,12 @@ def initialize(initial, can_be_empty)
       @sb = StringIO.new(initial)
     end
 
+    attr_accessor :can_be_empty
+
+    def can_be_empty?
+      @can_be_empty
```

Comment:
`attr_accessor` lets you directly access an instance variable, so you shouldn't need this `can_be_empty` method. With an instance of Element, you should just be able to call `elementInstance.can_be_empty` or `elementInstance.can_be_empty = false`.